src/kirby/components/empty-state/empty-state.component.scss

```diff
@@ -18,7 +18,7 @@ h3,p {
 }
 
 kirby-icon + * {
-  margin-top: size('m');
+  margin-top: size('s');
```

Comment:
Consider applying ` margin-bottom: size('s');` to `kirby-icon` instead unless you have a really good use case for doing the opposite :)

Reply:
The reason we add it as `margin-top` this way, is that the distance between the elements is controlled by the above element.

Reply:
Fixed